lasagne/updates.py

```diff
@@ -53,7 +54,7 @@
 
 import theano
 import theano.tensor as T
-
+import utils
```

Comment:
Travis is failling only for Python 3, and it looks like it's because this should be a relative import. I think it should read `from . import utils` (not tested).